src/database-redis.cpp

```diff
@@ -127,18 +129,14 @@ bool Database_Redis::deleteBlock(v3s16 blockpos)
 {
 	std::string tmp = i64tos(getBlockAsInteger(blockpos));
 
-	redisReply *reply = (redisReply *)redisCommand(ctx, "HDEL %s %s",
-		hash.c_str(), tmp.c_str());
+	redisReply *reply = static_cast<redisReply *>(redisCommand(ctx,
+		"HDEL %s %s", hash.c_str(), tmp.c_str()));
```

Comment:
Not that it matters, there should be two levels of indentation from `redisReply` here.